package.json

```diff
@@ -28,6 +28,7 @@
     "jest": "^26.6.3",
     "rimraf": "^3.0.2",
     "ts-jest": "^26.5.6",
+    "ts-node": "^9.1.1",
```

Comment:
ts-node shouldn't be needed. Check the package.json scripts and you can see that to compile the .ts files you can use the `build` script, which will output the compiled .js files to the `lib/` directory.

Reply:
It was required to build the code on my machine... Could it be installed globally on yours? Anyway, consider it gone :)

Reply:
The only thing I have installed globally is yarn. After running `yarn install` you should be able to run `yarn build` or `yarn build:watch` to compile the typescript source into the `lib/` directory.

Reply:
Yup, you are right, I have removed locally and globally, and everything still works!